wiki/Tournaments/UKCC/UKCC7/en.md

```diff
@@ -0,0 +1,636 @@
+  - Double Elimination Bracket
+- To be able to participate, players must have an **UK or Irish flag or be able to provide proof of residency to the hosts.**
+- Matches will be **Head to Head** and **Score V2**
+- Ranks and Qualifications placements are used to calculate seeding. Once the tournament starts, your rank is locked and will not affect your seed.
+- All staff (apart from the Hosts, Mappool Selectors and Referees) are allowed to participate in the tournament.
+- All players are to treat staff and other participants with a high level of respect. Any kind of mistreatment on the discord server or during matches (this includes homophobia, racism and toxicity) will result in a **BAN** and will be **BLACKLISTED** from participating from future UKCC tournaments.
+- ***Where rules do not prevail, common sense shall. The administration has explicit discretion to apply their judgement on this as they see fit.***
+- Players are **REQUIRED** to join the public discord so that they are contactable by other players and staff. If a player can’t be contacted or is not responding to messages, through discord, ingame messages or forum pms, contact the staff.
+- Mappools are to be released the Monday before a stage is due to happen
+
+### Reschedules
+
+- Schedules for the next round are released on the Monday before a stage is due to happen and **ALL** matches will be scheduled for the weekend. You can reschedule for weekdays at the staffs discretion.
+- In order to reschedule, you must contact your opponents, agree on a time and be able to provide **PROOF** (by form of a screenshot/image) of the agreed reschedule to the Hosts.
+- In effort to ensure as many matches take place as possible, reschedules are allowed up to **1 HOUR BEFORE YOUR MATCH IS DUE TO BE PLAYED**. However, your opponent holds the right to **refuse a reschedule if asked less than 24 hours from the scheduled time**
+- If it is impossible to find a time to reschedule to that suits both players, both players should talk to the Hosts (Bae- or Yazzehh) and we will attempt to sort something out.
+- Matches that are uncertain to happen (for example, losers’ bracket matches) will still appear on the schedule. They are to be rescheduled if needed in the same way as confirmed matches.
+- If a match needs rescheduling to a time after a related losers’ bracket match, the matches relying on the outcome of the rescheduled match will be rescheduled by staff and 24 hours will be given for rescheduling them, if needed.
+
+## Links
+
+![Links](img/links.png)
+
+- [Discussion Thread](https://osu.ppy.sh/community/forums/topics/940042)
+- [UKCC Discord server](https://discord.gg/EtsU7Cw)
+- [Livestream](https://twitch.tv/osuukcc)
+- [Challonge bracket](https://challonge.com/ukcc7)
+- [Pick'em predictions website](https://pickem.hwchr.com/tournaments/15)
+- **[Statistics sheet](https://docs.google.com/spreadsheets/d/1ubY1Oci0HaH-rny6IKbqlRszPzquwSm9n4fwwnqn5yA/edit?usp=sharing)**
+
+## Rules
```

Comment:
rules are usually put as the last section in tournament articles since they get pretty long and the only people who would rly care about this are the participants

Reply:
The images we made for our forum post are numbered as you can see:

![rules](https://user-images.githubusercontent.com/31848108/76759264-3b361c00-6783-11ea-8b72-fc9a40c12a82.png)

Would it be okay to overlook this for this reason?

Reply:
is it not possible to renumber them here

Reply:
I didn't make the graphics 😢 so not really. It's not the end of the world if it's out of order.